docs/src/Groups/groups.md

```diff
@@ -22,6 +22,138 @@ Julia supports the following types of groups:
 * `DirectProductOfGroups` = direct product of two groups
 * `AutomorphismGroup` = group of automorphisms over a group
 
-## Permutations groups
 
-In Julia, permutations are displayed as products of disjoint cycles, as in GAP.
+If `x` is an element of the group `G` of type `T`, then the type of `x` is `GAPGroupElement{T}`.
+
+## Basics
+
+### Elements of groups
+
+Given a group `G`, it is always possible to have access to some particular elements.
+
+* `one(G)` = the identity element of `G`.
+```@repl oscar
+one(symmetric_group(4))
+one(GL(2,3))
+```
+
+* `rand(G)`: returns a random element of the group `G`.
+
+For very large groups, the algorithm to ensure a uniform probability could be expensive, so there exists also the function `rand_pseudo`: this is much faster than `rand`, but the elements of `G` are not necessarily equally distributed.
+
+* `gens(G)`: returns a list of generators for the group `G`. To have access to the ``n``-th generator of the list, it is sufficient to type `G[n]` instead of `gens(G)[n]`.
+```@repl oscar
+G = symmetric_group(4);
+gens(G)
+G[1]
+```
+If the group `G` has been created as the subgroup of another group generated by a list of elements `L`, then the generating set returned by the function `gens` corresponds to `L`.
+
+!!! note
+    The output of `gens(G)` is not, in general, the minimal list of generators for `G`.
```

Comment:
Hmm.. I think this should be done differently: this text and the example should be part of the docstring for our methods for `one`, `rand`, gens`. Then here, we should just "include" that documentation via `@docs`

````suggestion
```@docs
one
rand
gens
```
````
Actually, that's probably not quite right yet, as `Oscar.jl` will include many `one` methods eventually, so we probably ought to tell it which one is meant here. Please carefully read https://juliadocs.github.io/Documenter.jl/stable/man/syntax/ it explains how to do this.

Perhaps also look at how some of our other packages do this; here are some select examples (you can easily find plenty more)
- https://github.com/oscar-system/GAP.jl/blob/master/docs/src/index.md 
- https://github.com/Nemocas/AbstractAlgebra.jl/blob/master/docs/src/perm.md
- https://github.com/thofma/Hecke.jl/blob/master/docs/src/number_fields/elements.md
- https://github.com/Nemocas/Nemo.jl/blob/master/docs/src/polynomial.md
Of course if you `gens` method has no good documentation comment right now, then you can take the text you wrote above and use that to form a doc comment instead.